papers/ice_nucleation_2024/calibration_setup.jl

```diff
@@ -133,7 +133,7 @@ function perf_model_pseudo_data(FT, IN_mode, params, IC)
     G_truth = run_model(params, coeff_true, IN_mode, FT, IC)
     dim_output = length(G_truth)
 
-    Γ = 0.03 * LinearAlgebra.I * (maximum(G_truth) - minimum(G_truth))
+    Γ = 0.001 * LinearAlgebra.I * (maximum(G_truth) - minimum(G_truth))
```

Comment:
How do we obtain this number?

Reply:
Need to ask AIDA people what the noise in their data is. I am using an arbitrary number here for now